qualang_tools/results/data_handler/data_processors.py

```diff
@@ -186,4 +186,9 @@ def post_process(self, data_folder: Path):
                 array.to_netcdf(data_folder / path.with_suffix(self.file_suffix))
 
 
-DEFAULT_DATA_PROCESSORS.append(XarraySaver)
+try:
```

Comment:
I don't think this is the right solution... I believe this part should be in the init file